src/functions/utils.php

```diff
@@ -25,3 +25,36 @@ function tribe_array_merge_recursive( array &$array1, array &$array2 ) {
 		return $merged;
 	}
 }
+
+if ( ! function_exists( 'tribe_can_plugin_run' ) ) {
+	/**
+	 * Checks if this plugin has permission to run, if not it notifies the admin
+	 *
+	 * @param string $file_path   Full file path to the base plugin file
+	 * @param string $main_class  The Main/base class for this plugin
+	 * @param string $version     The version
+	 * @param array  $classes_req Any Main class files/tribe plugins required for this to run
+	 *
+	 * @return bool Indicates if plugin should continue initialization
+	 */
+	function tribe_can_plugin_run( $file_path, $main_class, $version, $classes_req = array() ) {
+		$tribe_plugins = Tribe__Plugin_Dependencies::instance();
+
+		if ( $tribe_plugins->has_requisite_plugins( $classes_req ) ) {
+			$tribe_plugins->add_active_plugin( $main_class, $version, $file_path );
+
+			return true;
+		} elseif ( is_admin() ) {
+			$tribe_plugins = new Tribe__Plugins_List();
+			$admin_notice  = new Tribe__Plugin_Download_Notice( $file_path );
+
+			foreach ( $classes_req as $class => $version ) {
```

Comment:
That is intentional, just because it's what our current plugin notices do. I had considered maybe bolding ones that were inactive or some other visual trick. I might leave enhancements like this to after we move over to tribe_notice() in 4.3.